easybuild/easyblocks/generic/cargo.py

```diff
@@ -140,6 +148,8 @@ def get_checksum(src, log):
 class Cargo(ExtensionEasyBlock):
     """Support for installing Cargo packages (Rust)"""
 
+    BRANCH_MISSING = object()  # Used when branch information was not specified
```

Comment:
Same question as before, can we treat crates with a missing branch in the same as crates with branch=None? I would prefer this to be optional, so not mandatory to be specified.